src/Cubes/Axes.jl

```diff
@@ -293,7 +293,10 @@ Given the string of an axis name and a cube, returns this axis of the cube.
 getAxis(desc, c) = getAxis(desc, caxes(c))
 getAxis(desc::ByValue, axlist::Vector{T}) where {T<:CubeAxis} = desc.v
 
-"Fallback method"
+"""
+    findAxis(desc, c)
+Given an Axis description and a cube return the index of the Axis.
```

Comment:
Maybe we could quickly summarize what an axis description can be (Axis name, abbreviated axis name, an axis itself...)? Would be more clear

Reply:
I started a list. 
Could you explain how the ByFunction description is used?
Because I haven't found an example and haven't used it myself yet.

Reply:
I think we can remove this one. It was necessary to have when we were still registering functions, but it is obsolete now.

Reply:
I mean we can remove everything connected to "ByFunction"

Reply:
I'll open a new PR for that.

Reply:
See #108